Dockerfile

```diff
@@ -65,3 +62,4 @@ ENV DEBUG ${SERVER_DEBUG}
 
 COPY docker-entrypoint.sh /
 ENTRYPOINT ["/docker-entrypoint.sh"]
+#ENTRYPOINT ["bash"]
```

Comment:
Remove this line. I guess it was for debug

Reply:
Yes, it was just for debugging. Removed